rest_framework-stubs/fields.pyi

```diff
@@ -29,10 +31,15 @@ from django.core.files.base import File
 from django.db import models
 from django.db.models import Model
 from django.forms import ImageField as DjangoImageField  # noqa: F401
-from rest_framework.serializers import BaseSerializer
 from typing_extensions import Literal
 
-class empty: ...
+from rest_framework.serializers import BaseSerializer
+
+class _Empty(Enum):
+    sentinel = 0
+
+empty: Final = _Empty.sentinel
```

Comment:
Rationale https://github.com/typeddjango/djangorestframework-stubs/issues/42

Reply:
interesting. did you raise this with the DRF folk?

Reply:
No, because django core devs are very conservative about adding typings to their codebase also DRF right now is mostly in bugfixes state, so I don't have enough willpower to push that change.